### ISSUE TITLE
Defias should be able to use the raid mechanic.

### ISSUE THREAD

**Issue by geffe1** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#1515]
**Describe your suggestion in full detail below:**
I believe that the Defias should be able to loot neighbours, perhaps a way to make this possible is to give them a unique culture- or religion.

I think it would make them more lore-accurate, allowing them to loot. As they did many such things against Stormwind. 

>The Stonemasons subsequently became bandits, collecting their promised payment by robbing travelers. Banditry and thievery quickly became a way of life for the exiles, who banded together under VanCleef's leadership as the Defias Brotherhood. 

https://wow.gamepedia.com/Defias_Brotherhood
![ck2_529](https://user-images.githubusercontent.com/31706375/80769346-31a71e80-8b4d-11ea-8992-26da5ed2a662.png)

**Comment by zumbak04** [comment]
Agree. We had an idea to create a special government form for them.
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/projects/57
